### ISSUE TITLE
Error when selecting pulses using boolean array for ds.plot_summaries

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#213]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/213) by Charles James Titus (Bitbucket: [cjtitus](https://bitbucket.org/cjtitus), GitHub: [cjtitus](https://github.com/cjtitus)).**

----------------------------------------

ds.plot\_summaries claims that you can pass in a boolean array for the “valid” argument

However, this throws an error on line [https://bitbucket.org/joe\_fowler/mass/src/df0e1254a8ebf260d7f6c045489c3fbf72451657/mass/core/channel.py#lines-1553](https://bitbucket.org/joe_fowler/mass/src/df0e1254a8ebf260d7f6c045489c3fbf72451657/mass/core/channel.py#lines-1553)

```
~/work/mass/mass/core/channel.py in plot_summaries(self, valid, downsample, log)
1551                     downsample = 1
1552             hour = self.p_timestamp[::downsample] / 3600.0
-> 1553         LOG.info("%s (%d records; %d in scatter plots)", status, nrecs, len(hour))
1554
1555         plottables = (
UnboundLocalError: local variable ‘status' referenced before assignment
```

Because `status` is only set when `valid` is a string.

Status should get set to something when `valid` is a boolean array.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Fix error on ds.plot_summaries(valid=B)

Fixes #213.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.7 (automated comment)
